cmd/gitbase/command/server.go

```diff
@@ -57,6 +60,7 @@ type Server struct {
 	Parallelism   uint           `long:"parallelism" description:"Maximum number of parallel threads per table. By default, it's the number of CPU cores. 0 means default, 1 means disabled."`
 	DisableSquash bool           `long:"no-squash" description:"Disables the table squashing."`
 	TraceEnabled  bool           `long:"trace" env:"GITBASE_TRACE" description:"Enables jaeger tracing"`
+	MetricsPort   int            `long:"metrics-port" default:"2112" description:"Port where the server is going to expose Prometheus metrics"`
```

Comment:
some environment variable for that too?

Reply:
If you want to start it on different port. You have to pass an address to Prometheus to know from what endpoint pull metrics.